libraries/standard/http/include/http_client.h

```diff
@@ -0,0 +1,339 @@
+#ifndef HTTP_CLIENT_H_
+#define HTTP_CLIENT_H_
+
+#include <stdint.h>
+#include <stdbool.h>
+
+/**
+ * @brief Maximum amount of headers allowed from the server.
+ * 
+ * If the server sends more headers than this configuration, then status code
+ * #HTTP_SECURITY_ALERT_TOO_MANY_HEADERS is returned from #HTTPClient_Send.
+ */
+#ifndef HTTP_MAX_HEADERS_BYTES
+    #define HTTP_MAX_HEADERS_BYTES      2048U
+#endif
+
+/**
+ * @brief The HTTP header "User-Agent" value.
+ * 
+ * The following headerline is automatically written to
+ * #HTTPRequestHeaders_t.pBuffer:
+ * "User-Agent: AWS\r\n"
+ */ 
+#ifndef HTTP_USER_AGENT_VALUE
+    #define HTTP_USER_AGENT_VALUE       "AWS"
+#endif
+
+/**
+ * Supported HTTP request methods.
+ */
+#define HTTP_METHOD_GET                 "GET"  /**< HTTP Method GET. */
+#define HTTP_METHOD_PUT                 "PUT"  /**< HTTP Method GET. */
+#define HTTP_METHOD_POST                "POST" /**< HTTP Method GET. */
+#define HTTP_METHOD_HEAD                "HEAD" /**< HTTP Method GET. */
+
+
+/**
+ * Flags for #HTTPRequestInfo_t.flags.
+ * These headers control what headers are written or not to the 
```

Comment:
Do you mean "These _flags_ ~~headers~~ control what..."?

Reply:
Yes thanks